types/mapcmd/map.go

```diff
@@ -256,7 +256,7 @@ func decorateCommandWithMapKeyTypeFlags(cmd *cobra.Command, mapKeyType *string,
 }
 
 func decorateCommandWithMapValueTypeFlags(cmd *cobra.Command, mapValueType *string, required bool) {
-	cmd.Flags().StringVarP(mapValueType, MapValueTypeFlag, MapValueTypeFlagShort, "string", fmt.Sprintf("type of the value, one of: %s", strings.Join(internal.SupportedTypeNames, ",")))
+	cmd.Flags().StringVarP(mapValueType, MapValueTypeFlag, MapValueTypeFlagShort, "", fmt.Sprintf("type of the value, one of: %s", strings.Join(internal.SupportedTypeNames, ",")))
```

Comment:
How about "value type" instead of "type of the value" to make it shorter?

Reply:
https://github.com/hazelcast/hazelcast-commandline-client/pull/114/commits/33e5d0a48d545a710b929043f671ab0808485bcf